jenkins/bootstrap.py

```diff
@@ -883,7 +883,10 @@ def bootstrap(args):
             start(gsutil, paths, started, node(), version, repos)
         success = False
         try:
-            call(job_script(job, use_json))
+            job_cmd = job_script(job, use_json)
+            if upload:
+                job_cmd += ['--gcs-artifacts-dir=%s' % paths.artifacts]
```

Comment:
The only thing that changes between runs is the `BUILD_NUMBER` env and that is accessible to you between runs. I recommend using that and setting the prefix somewhere in jobs/config.json for the jobs that want to use it.

Ideally we pass this along in a more explicit way but injecting a flag into the run command will break existing usage. This is probably not what we want to do...

I would say update scenarios/kubernetes_e2e.py to consume the BUILD_NUMBER env (and leave it to Sen or I to refactor this into something else later)